build.gradle

```diff
@@ -14,14 +14,34 @@ repositories {
     maven { url 'http://repo.jenkins-ci.org/releases/' }
 }
 
+configurations {
+    testPlugins {}
+}
+
 dependencies {
     compile 'org.codehaus.groovy:groovy:2.1.3'
-    compile 'org.jenkins-ci.plugins:job-dsl-core:1.47'
+    compile "org.jenkins-ci.plugins:job-dsl-core:${jobDslVersion}"
+
     testCompile('org.spockframework:spock-core:0.7-groovy-2.0') {
         exclude module: 'groovy-all'
     }
     testCompile 'cglib:cglib-nodep:2.2.2' // used by Spock
 
+    // Jenkins test harness dependencies
+    testCompile 'org.jenkins-ci.main:jenkins-test-harness:2.8'
+    testCompile "org.jenkins-ci.main:jenkins-war:${jenkinsVersion}"
+    testCompile "org.jenkins-ci.main:jenkins-war:${jenkinsVersion}:war-for-test@jar"
+
+    // Job DSL plugin including plugin dependencies
+    testCompile "org.jenkins-ci.plugins:job-dsl:${jobDslVersion}"
+    testCompile "org.jenkins-ci.plugins:job-dsl:${jobDslVersion}@jar"
+    testCompile 'org.jenkins-ci.plugins:structs:1.1@jar'
+    testCompile 'org.jenkins-ci.plugins:cloudbees-folder:4.4@jar'
+
+    // plugins to install in test instance
+    testPlugins 'org.jenkins-ci.plugins:ghprb:1.31.4'
```

Comment:
can this resolve the issue with Generated DSL method? I tried to follow the configuration here, but still got the issue
```
javaposse.jobdsl.dsl.DslScriptException: (PullRequestJobTemplate.groovy, line 59) No signature of method: javaposse.jobdsl.dsl.helpers.triggers.TriggerContext.githubPullRequest() is applicable for argument types: (com.unified.dsl.templates.PullRequestJobTemplate$_closure1$_closure5$_closure15) values: [com.unified.dsl.templates.PullRequestJobTemplate$_closure1$_closure5$_closure15@40883107]
```

Reply:
the method `githubPullRequest` is provided by `ghprb` plugin, I tried to at the plugin into `testPlugins`, but the unit test still got failure